### PR TITLE
Make node related setting names plural

### LINKED ISSUE #262: Rename `setting` to `settings` in `Node`
This is for clarity and consistency.

### PR DESCRIPTION
**Pull request by sophie-cluml**
closes #262 

---

Notes
- Migration code is not necessary for this change, because name change does not affect serialization layout.
- I incremented the pre-release alpha version by 1, as advised by Sehkone.

### CODE REVIEW

File: src/tables/node.rs [hunk 1]
```diff
@@ -69,8 +69,8 @@ pub struct Node {
     pub id: u32,
     pub name: String,
     pub name_draft: Option<String>,
-    pub setting: Option<Setting>,
-    pub setting_draft: Option<Setting>,
+    pub settings: Option<Settings>,
+    pub setting_draft: Option<Settings>,
```
Comment (minshao) [inline src/tables/node.rs:73]: `settings_draft`?

File: src/tables/node.rs [hunk 2]
```diff
@@ -130,16 +130,16 @@ impl<'d> IndexedTable<'d, Node> {
 pub struct Update {
     pub name: Option<String>,
     pub name_draft: Option<String>,
-    pub setting: Option<Setting>,
-    pub setting_draft: Option<Setting>,
+    pub setting: Option<Settings>,
+    pub setting_draft: Option<Settings>,
```
Comment (minshao) [inline src/tables/node.rs:134]:
```suggestion
    pub settings: Option<Settings>,
    pub settings_draft: Option<Settings>,
```

Comment (sophie-cluml) [reply src/tables/node.rs]: done

File: src/tables/node.rs (hunk 1 above)
Comment (sophie-cluml) [reply src/tables/node.rs]: done

**Review by minshao — APPROVED** [review]

**Pull request merged**